<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileNamePrompt</class>
 <widget class="QWidget" name="FileNamePrompt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>123</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>380</width>
    <height>123</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>380</width>
    <height>123</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Save</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>20</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>File name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_filename">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>18</y>
     <width>301</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_save">
   <property name="geometry">
    <rect>
     <x>308</x>
     <y>95</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_cancel">
   <property name="geometry">
    <rect>
     <x>241</x>
     <y>95</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_location">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
